DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -619,10 +618,29 @@ DasharoSystemFeaturesCallback (
 {
   EFI_STATUS                                 Status;
   EFI_INPUT_KEY                              Key;
+  DASHARO_SYSTEM_FEATURES_PRIVATE_DATA       *Private;
 
   Status = EFI_SUCCESS;
+  Private = DASHARO_SYSTEM_FEATURES_PRIVATE_DATA_FROM_THIS (This);

```

Comment:
CI complains that this isn't used.  `PRIVATE_DATA()` uses `mDasharoSystemFeaturesPrivate` directly.

Reply:
Right, I should change the access to CpuThrottlingOffset to use `Private` pointer.

Reply:
Fixed: https://github.com/Dasharo/edk2/compare/e149e1ee6e13b71ed16a30aa27261d4c69eca3fe..27e4a00a26c5780ea53f8274f55c6dedb5496657